src/assets/locales/es.json

```diff
@@ -173,288 +173,288 @@
-  "tips.mobile.pen_mode": "Vuelve a tocar el icono para finalizar el dibujo",
-  "tips.mobile.mute_mode": "Estás silenciado. Presiona para dejar de silenciar",
-  "tips.mobile.freeze_mode": "Toca con dos dedos para ocultar los menús",
-  "tips.mobile.look": "¡Bienvenido! 👋 Toca y arrastra para mirar a tu alrededor.",
-  "tips.mobile.locomotion": "¡Genial! Para moverse, pellizque con dos dedos.",
-  "tips.mobile.spawn_menu-pre": "Utiliza el ",
-  "tips.mobile.spawn_menu-post": "en la parte superior para crear objetos.",
-  "tips.mobile.freeze_gesture": "Toca con dos dedos para mostrar los menús.",
-  "tips.mobile.object_grab": "Arrastra un objeto para moverlo o lanzarlo",
+  "tips.mobile.video_share_mode": "Toca para dejar de compartir video.",
+  "tips.mobile.pen_mode": "Toca nuevamente el ícono para dejar de dibujar.",
+  "tips.mobile.mute_mode": "Estás silenciado(a). Toca para activar el micrófono.",
+  "tips.mobile.freeze_mode": "Toca con dos dedos para ocultar los menús.",
+  "tips.mobile.look": "¡Bienvenido(a)! 👋 Toca y arrastra para mirar alrededor.",
+  "tips.mobile.locomotion": "¡Genial! Para moverte, pellizca con dos dedos.",
+  "tips.mobile.spawn_menu-pre": "Usa ",
+  "tips.mobile.spawn_menu-post": "arriba para crear objetos.",
+  "tips.mobile.freeze_gesture": "Toca con dos dedos para el menú contextual.",
+  "tips.mobile.object_grab": "Arrastra un objeto para moverlo o tirarlo.",
   "tips.mobile.object_rotate_button-pre": "Toca y arrastra ",
-  "tips.mobile.object_rotate_button-post": " para rotar el objeto.",
+  "tips.mobile.object_rotate_button-post": " para girar.",
   "tips.mobile.object_scale_button-pre": "Toca y arrastra ",
-  "tips.mobile.object_scale_button-post": " para escalar el objeto.",
+  "tips.mobile.object_scale_button-post": " para cambiar la escala.",
   "tips.mobile.object_recenter_button-pre": "Toca ",
-  "tips.mobile.object_recenter_button-post": " para que el objeto esté frente a ti.",
-  "tips.mobile.object_pin": "Fijar un objeto para guardarlo en la habitación.",
-  "tips.mobile.video_share_failed": "No se ha concedido permiso.",
-  "tips.mobile.invite": "Use el botón Compartir para compartir esta sala.",
-  "tips.mobile.feedback": "🦆 ¡Quack! ¿Quieres ayudar a Hubs?",
-  "tips.mobile.feedback-link": "Dile al pato.",
-  "tips.desktop.video_share_mode": "Estás transmitiendo. Selecciona nuevamente para dejar de compartir videos.",
+  "tips.mobile.object_recenter_button-post": " para hacer que el objeto mire hacia ti.",
+  "tips.mobile.object_pin": "Fija un objeto para guardarlo dentro de la sala.",
+  "tips.mobile.video_share_failed": "No hay permiso otorgado.",
+  "tips.mobile.invite": "Usa el botón Compartir para compartir esta sala.",
+  "tips.mobile.feedback": "🦆 ¡Cuac! ¿Quieres ayudar a Hubs?",
+  "tips.mobile.feedback-link": "Cuéntale al pato.",
+  "tips.desktop.video_share_mode": "Estás haciendo una transmisión. Selecciona nuevamente para dejar de compartir el video.",
```

Comment:
"Estás haciendo una transmisión. Selecciona nuevamente para dejar de compartir el video." should be "Estás transmitiendo. Selecciona nuevamente para dejar de compartir el video."